swagger-templates/python/rest.mustache

```diff
@@ -0,0 +1,252 @@
+# coding: utf-8
+
+{{>partial_header}}
+from __future__ import absolute_import
+
+import sys
+import io
+import json
+import ssl
+import certifi
+import logging
+
+# python 2 and python 3 compatibility library
+from six import iteritems
+
+from .configuration import Configuration
+
+try:
+    import urllib3
+except ImportError:
+    raise ImportError('Swagger python client requires urllib3.')
+
+try:
+    # for python3
+    from urllib.parse import urlencode
+except ImportError:
+    # for python2
+    from urllib import urlencode
+
+
+logger = logging.getLogger(__name__)
+
+
+class RESTResponse(io.IOBase):
+
+    def __init__(self, resp, method):
+        self.urllib3_response = resp
+        self.status = resp.status
+        self.reason = resp.reason
+        # In Square Connect v2 API, 
+        # GET ListCustomers/ListLocations/ListTransaction/ListRefunds
+        # may have large response thus calling stream() for chunked-encoding
+        if method in ['GET', 'HEAD']:
```

Comment:
Maybe make this more general? A response is chunked only if there is an HTTP header that indicates this: `Transfer-Encoding: chunked`